zkevm-circuits/src/witness/block.rs

```diff
@@ -156,46 +156,50 @@ pub struct BlockContext {
 
 impl BlockContext {
     /// Assignments for block table
-    pub fn table_assignments<F: Field>(&self, randomness: Value<F>) -> Vec<[Value<F>; 3]> {
+    pub fn table_assignments<F: Field>(&self, _randomness: Value<F>) -> Vec<[Value<F>; 4]> {
```

Comment:
The randomness can go
```suggestion
    pub fn table_assignments<F: Field>(&self) -> Vec<[Value<F>; 4]> {
```

Reply:
Have clean up the _randomness var in commit https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1411/commits/0646b790010306d9df41cd27002bf9890d017202